workers/unity/Assets/Fps/Scripts/GameLogic/StateMachine/Default/DefaultState.cs

```diff
@@ -18,7 +18,16 @@ protected DefaultState(UIManager manager, ConnectionStateMachine owner)
             Owner = owner;
             Blackboard = Owner.Blackboard;
 
-            Animator = manager.GetComponentInChildren<Animator>(true);
+            Animator = manager.ScreenManager.DefaultConnectButton.GetComponent<Animator>();
+            Debug.Log(GetType());
```

Comment:
Debug log

Reply:
ah yeah, forgot to take all the debug statements out